frontend/website/src/Testnet.re

```diff
@@ -256,12 +421,126 @@ module Section = {
     </div>;
   };
 };
+let f = Links.Cdn.url;
 
 [@react.component]
 let make = () => {
   <div className=Styles.page>
+    <div className=Styles.heroRow>
+      <div className=Styles.heroText>
+        <h1 className=Style.H1.hero>
+          {React.string("Coda Public Testnet")}
+        </h1>
+        <p className=Style.Body.basic>
+          {React.string(
+             "Coda's public testnet is live! There are weekly challenges for the community \
+                  to interact with the testnet and contribute to Coda's development. Each week \
+                  features a new competition to recognize and reward top contributors with testnet \
+                  points.",
+           )}
+        </p>
+        <br />
+        <p className=Style.Body.basic>
+          {React.string(
+             "By participating in the testnet, you'll be helping advance the first cryptocurrency that utilizes recursive zk-SNARKs and production-scale Ouroboros proof of stake consensus.",
+           )}
+        </p>
+      </div>
+      <div id="termynal" className=Styles.termynal>
+        <RunScript>
+          {|var termynal = new Termynal('#termynal', {
+            typeDelay: 40,
+            lineDelay: 700,
+            lineData: [
+              { type: 'input', prompt: '>', value: 'coda daemon -peer ...' },
+              { type: 'progress' },
+              { value:  'Daemon ready. Clients can now connect!'},
+              { type: 'input', prompt: '>', value: 'coda client status' },
+              { value:  'Local uptime: 25m25s'},
+              { value:  'Peers: 5'},
+              { value:  'Consensus time now: epoch=16, slot=78'},
+              { value:  'Sync status: Synced'},
+            ]
+          });|}
+        </RunScript>
+      </div>
+    </div>
+    <div>
+      <div className=Styles.buttonRow>
```

Comment:
These buttons seem pretty similar, at some point it could be nice to build a little subcomponent for it.

```
<SocialButton icon=... title=... body=... href=... />
```
or whatever